runtime/ext.rs

```diff
@@ -27,11 +30,66 @@ impl TimersPermission for ZinniaPermissions {
 }
 
 impl FetchPermissions for ZinniaPermissions {
-    fn check_net_url(&mut self, _url: &Url, _api_name: &str) -> Result<()> {
+    fn check_net_url(&mut self, _url: &Url, _api_name: &str) -> Result<(), PermissionCheckError> {
         Ok(())
     }
-    fn check_read(&mut self, _p: &Path, _api_name: &str) -> Result<()> {
-        Ok(())
+    fn check_read<'a>(
+        &mut self,
+        _resolved: bool,
+        _p: &'a Path,
+        _api_name: &str,
+    ) -> Result<Cow<'a, Path>, FsError> {
+        Err(FsError::NotCapable("read local filesystem"))
+    }
+}
+
+impl NetPermissions for ZinniaPermissions {
+    fn check_net<T: AsRef<str>>(
```

Comment:
Will the user running checker app and or zinnia runtime have to grant these permissions to the app?

Reply:
No, the Deno Net API operations are not permitted in Zinnia. 

At the moment, Zinnia modules are allowed to use the `fetch()` API only, which uses a different permission check.

In the future, if we decide to expose Deno Net APIs, then we can change the implementation of this permission check.